tests/Earthfile

```diff
@@ -565,6 +566,11 @@ fail-invalid-artifact-test:
     DO +RUN_EARTHLY --earthfile=fail-invalid-artifact.earth --should_fail=true --target="--artifact +test/foo /tmp/stuff" \
         --output_contains="cannot save artifact +test/foo, since it does not exist"
 
+wildcard-build:
+    COPY --dir wildcard-build .
+    DO +RUN_EARTHLY --earthfile=wildcard-build.earth --target=+wildcard-build --output_does_not_contain="not-test run"
```

Comment:
can you add some extra tests here to verify `earthly.output` (which is created by `DO +RUN_EARTHLY`) contains the expected "hello world" strings.

Reply:
Thanks. I forgot about that output file.